src/test/java/com/metamx/common/io/smoosh/SmooshedFileMapperTest.java

```diff
@@ -71,6 +71,142 @@ public void testSanity() throws Exception
       }
     }
   }
+  
+  @Test
+  public void testSanityForMultipleSmooshedFileMapperOpened() throws Exception
```

Comment:
I'm not sure I understand what this is testing, it looks to be doing the same thing that `testSanity` is doing.  Did you intend for it to immediately open a writer and then close it after the loop is complete?